filebeat/fileset/pipelines.go

```diff
@@ -58,6 +58,13 @@ func (reg *ModuleRegistry) LoadPipelines(esClient PipelineLoader, overwrite bool
 				return fmt.Errorf("Error getting pipeline for fileset %s/%s: %v", module, name, err)
 			}
 
+			// Filesets with multiple pipelines can only be supported by Elasticsearch >= 6.5.0
+			esVersion := esClient.GetVersion()
+			minESVersionRequired := common.MustNewVersion("6.5.0")
+			if len(pipelines) > 1 && esVersion.LessThan(minESVersionRequired) {
```

Comment:
Does the current LS module also fall into this statement? Probably not because it only loads 1?

Reply:
The Logstash module actually loads both/all pipelines but it only uses one of them at runtime. So it doesn't fall into this statement. I just tested this to confirm as well.

Reply:
But wouldn't we want to fail installing the LS module?

Reply:
I'm not sure. The `logstash/log` fileset's `manifest.yml` looks like this:

https://github.com/elastic/beats/blob/f5a902839c5f5b1031c3d904dc561bbb4239f9e0/filebeat/module/logstash/log/manifest.yml#L19

Whereas a fileset that is using the multiple pipelines feature will look like this:

```yml
ingest_pipeline:
  - ingest/entry_point_pipeline.json
  - ingest/some_sub_pipeline.json
  - ingest/another_sub_pipeline.json
```